devices/src/legacy/rtc_pl031.rs

```diff
@@ -265,7 +265,7 @@ mod tests {
         assert_eq!(no_errors_after - no_errors_before, 1);
         // However, reading from the AMBA_ID_LOW should succeed upon read.
 
-        let mut data = [0; 8];
+        let mut data = [0; 4];
```

Comment:
Please add some more details to the commit message so we know why we made it 4 instead of 8.